code/__DEFINES/access_defines.dm

```diff
@@ -65,7 +65,7 @@
 #define ACCESS_XENOARCH 65
 #define ACCESS_PARAMEDIC 66
 #define ACCESS_BLUESHIELD 67
-#define ACCESS_SALVAGE_CAPTAIN 69 // Salvage ship captain's quarters
+// #define ACCESS_SALVAGE_CAPTAIN 69 // Salvage ship captain's quarters
```

Comment:
Could probably just remove it as it's been unused

Reply:
Done